lux/test/lux/storage/schemas/job_opening_test.exs

```diff
@@ -0,0 +1,103 @@
+defmodule Lux.Storage.Schemas.JobOpeningTest do
+  use ExUnit.Case, async: true
+  alias Lux.Storage
+  alias Lux.Storage.Schemas.JobOpening
+  alias Lux.Repo
+
+  setup do
+    # Start the sandbox for this test process
+    Ecto.Adapters.SQL.Sandbox.mode(Lux.Repo, :manual)
```

Comment:
Usually we inlcude this sandbox code in a module called DataCase, and use that as the test template that loads all helpers and what not.